tests/integ/test_trial_component.py

```diff
@@ -23,6 +23,10 @@ def test_create_delete(trial_component_obj):
     assert trial_component_obj.trial_component_name
 
 
+def test_create_tags(trial_component_obj):
+    assert trial_component_obj.tags
```

Comment:
this looks like this is just testing that the test TC has the tags populated. instead add tags to the existing test_save integ test here. same thing in trial and experiment integ tests.

Reply:
updated the integ test with actual tag check (calling sagemaker_boto_client.list_tags() check)